src/Thruway/Role/Subscriber.php

```diff
@@ -128,6 +134,37 @@ protected function processSubscribed(ClientSession $session, SubscribedMessage $
             }
         }
     }
+    
+    /**
+     * Process unsubscribe error
+     * 
+     * @param \Thruway\AbstractSession $session
+     * @param \Thruway\Message\ErrorMessage $msg
+     */
+    protected function processUnsubscribeError(AbstractSession $session, ErrorMessage $msg)
```

Comment:
`processUnsubscribeError` is correct. Only the broker would handle the `UNSUBSCRIBE` message.